{{cookiecutter.__src_folder_name}}/src/entrypoint.sh

```diff
@@ -1,15 +1,12 @@
 #!/bin/bash
-
 set -e
-
 python3 -m pip install --upgrade pip
 {% if cookiecutter.project_backend in ("flask", "fastapi") %}
 python3 -m pip install -e .
 {% endif %}
 {% if cookiecutter.project_backend == "flask" %}
```

Comment:
What happened to the migrations?

Reply:
good point I can look into - my guess is they were removed when working with mongoDB but not added back.